docker_config/compose_files/docker-compose.yml

```diff
@@ -236,7 +237,7 @@ services:
 
   frontend:
     image: codalab/frontend:${CODALAB_VERSION}
-    command: serve -s build -l ${CODALAB_FRONTEND_PORT}
+    command: bash -c "node setup-frontend.js && serve -s build -l ${CODALAB_FRONTEND_PORT}"
```

Comment:
By running `setup-frontend.js` we read react app environment variables from the runtime and allows the production website to read them.